.github/workflows/pylint-workflow.yml

```diff
@@ -0,0 +1,25 @@
+name: Linter Test
```

Comment:
`pylinting ` i think is not beneficial in our project, due to the fact that's most of the linting issues outputted not related to the code we wrote, which leads to some confusion to recognize which code belongs to us or not in the output; what you think?

Reply:
Yeah, we seem to be having a ton of import issues. We need to find a way of importing the Django modules into the linter test.